packages/popup/src/pages/Home/components/TokenList.js

```diff
@@ -0,0 +1,65 @@
+import {useEffect} from 'react'
+import PropTypes from 'prop-types'
+import {useTranslation} from 'react-i18next'
+import {PlusOutlined} from '@fluent-wallet/component-icons'
+import {formatBalance} from '@fluent-wallet/data-format'
+import {WrapIcon} from '../../../components'
+
+function TokenItem({
+  icon,
+  name = 'Conflux',
+  symbol = 'CFX',
+  balance = '1234567890.123456',
+}) {
+  useEffect(() => {
+    const balanceDom = document.getElementById('balance')
+    const contentWidth = balanceDom.offsetWidth
+    console.log(contentWidth)
+    if (contentWidth > 175) {
```

Comment:
better add magic number (175) in constant file.

Reply:
175 is used in css, so here is used direactly